.idea/runConfigurations.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
do not add any IDE specific files to the repo

Reply:
Hi, have added .idea folder itself in .gitignore which will excluded all IDE specific files to ad in repo.